recipes/gql/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "gql" %}
+{% set version = "0.1.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d20cf1eb1c6d54558174a3dcc3cf1ca0e07678cb1123804c2e165838238350f4" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six >=1.10.0
+    - graphql-core >=0.5.0
+    - promise >=0.4.0
+  run:
+    - python
+    - six >=1.10.0
+    - graphql-core >=0.5.0
+    - promise >=0.4.0
+
+test:
+  imports:
+    - gql
+    - gql.transport
+  requires:
```

Comment:
Why do these imports require mock and pytest?

Reply:
They were required in `setup.py` for unit testing (I created the recipe with `conda skeleton`). It builds also without.